### PR TITLE
New gamepad plugin

### PR DESCRIPTION
**Pull request by Shaobin-Jiang**
Using gamepads in experiments is sometimes needed, and while most browsers support it, a certain tweaking around is still needed. This plugin handles input from a gamepad and displays a minature model on the page that reflects user input in real time for the convenience of debugging.

### CODE REVIEW

File: packages/plugin-gamepad/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@Shaobin-Jiang/jspsych-gamepad",
```
Comment (jodeleeuw) [inline packages/plugin-gamepad/package.json:2]: To publish this in our npm namespace this should be `"@jspsych-contrib/plugin-gamepad"`

**Review by jodeleeuw — CHANGES_REQUESTED** [review]
Thanks for the contribution @Shaobin-Jiang! 

I left one small tweak to make to get the plugin to publish in the correct npm location.

**Comment by jodeleeuw** [comment]
Thank you @Shaobin-Jiang !

**Comment by jodeleeuw** [comment]
Reverted this because I missed the lack of a changeset. Will fix.

**Comment by jodeleeuw** [comment]
Sorry @Shaobin-Jiang -- can you make a new PR? I should've checked more carefully first and just added the changeset as a separate commit. Can you also target `@jspsych/config` v2 in `package.json`?

**Comment by bjoluc** [comment]
@jodeleeuw Technically, we can also add a changeset for a specific PR after merging it thanks to https://github.com/changesets/changesets/pull/535 :+1:

**Comment by jodeleeuw** [comment]
oops. good to know, thank you!

**Comment by Shaobin-Jiang** [comment]
@jodeleeuw Done in #86

**Pull request merged**